app/experimenter/experiments/migrations/0226_remove_viz_api_v1.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.14 on 2022-07-12 21:19
+
+from django.db import migrations
+
+
+def remove_viz_api_v1(apps, schema_editor):
+    NimbusExperiment = apps.get_model("experiments", "NimbusExperiment")
+
+    for experiment in NimbusExperiment.objects.all():
+        if experiment.results_data is not None and "v1" in experiment.results_data:
+            del experiment.results_data["v1"]
```

Comment:
I've always hated that this is the syntax to remove a key from a dictionary.  It's the only time you use a keyword like this, it should just be a method.  And I mean yeah you can just do experiment.results_data.pop('v1') and just ignore the return value, but I wish it was a little clearer.

Reply:
Also do we know for sure that every dictionary in the db has this key?  Should we check for its presence here?

Reply:
Oh nevermind you already tested for those cases 🎉 🎉 🎉 🎉

Reply:
Agreed, it's odd syntax for Python, but at least as far as uncommon syntax goes it's pretty obvious what it's doing.